src/Catalyst.Core.Modules.Web3/ApiModule.cs

```diff
@@ -86,25 +94,35 @@ async void BuildCallback(IContainer container)
             base.Load(builder);
         }
 
-        public void ConfigureServices(IServiceCollection services, ContainerBuilder containerBuilder)
+        //todo Clean up - find a better way to reuse the ContainerBuilder loaded into the module
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4403188)